scripts/test_downgrade_from_tag.sh

```diff
@@ -138,12 +138,12 @@ docker_run_vol() {
 
 wait_for_pg() {
     set +e
+    # shellcheck disable=SC2034
     for i in {1..20}; do
```

Comment:
```suggestion
    for _ in {1..20}; do
```

We can use underscore for dummy variables, shellcheck won't complain.

Reply:
Nice!